### PR TITLE
Write up for listing successes on AskTheEU homepage

### LINKED ISSUE #43: Does an automatically generated list of recent successful requests result in more people clicking on the start of the 'make a request' funnel than a curated set of 'top requests'?
## Q: Does an automatically generated list of recent successful requests result in more people clicking on the start of the 'make a request' funnel than a curated set of 'top requests'?

[Explain how you will measure this]

Measure change in conversion funnel

[What do we need to do to be able to take the measurements?]

Make version of front page that uses list of successful requests  - A/B test in google. Make sure funnel is defined in google analytics

### PR DESCRIPTION
**Pull request by lizconlan**
Closes #43

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,5 +223,18 @@ A collection of experiments and research to learn about what makes an Alaveteli
         <strong>Predominantly social media.</strong> Sites with a strong journalistic focus get a greater proportion of traffic from news and campaign sites.
       </td>
     </tr>
+    <tr>
+      <td>
+        <a href="experiments/effect-of-listing-successes-on-asktheeu-homepage.md">
+          We currently show a curated list of requests on the homepage, would
+          visitors be inspired to make their own requests if we showed them
+          recent successes instead?
```
Comment (garethrees) [inline README.md]: I'd be more terse: Does showing recent successful requests increase conversion rate?

File: README.md [hunk 2]
```diff
@@ -223,5 +223,18 @@ A collection of experiments and research to learn about what makes an Alaveteli
...
+        </a>
+      </td>
+      <td>
+        <strong>Probably not.</strong> There was no noticeable difference in the
+        rate of new requests between the 2 variations of the homepage.
```
Comment (garethrees) [inline README.md]: s/Probably not/No/ – We didn't see an increase. Also doesn't need the explainer IMO;

File: experiments/effect-of-listing-successes-on-asktheeu-homepage.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Can we inspire visitors to the AskTheEU site to make their own requests by
+showing them a list of recent successes instead of the current curated list
+of interesting requests?
+
+- [issue#43](https://github.com/mysociety/alaveteli-experiments/issues/43)
+
+## Answer
+
+**TL;DR:** Probably not.
```
Comment (garethrees) [inline experiments/effect-of-listing-successes-on-asktheeu-homepage.md]: I'd say the answer here is "No", because we didn't see an increase.

**Comment by garethrees** [comment]
Couple of minor tweaks then good to go.

File: README.md (hunk 2 above)
Comment (lizconlan) [inline README.md]: Fair enough

**Pull request merged**